.github/workflows/ui-release-trigger.yml

```diff
@@ -0,0 +1,20 @@
+name: "Trigger Boundary UI release"
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Do we really want to release for any tag?  This would preclude the use of tags for any other purpose.

Reply:
Tagging @99 for visibility.  I agree with @shore.  For the time being let's use what's here.  We'd like to push out a release today.

Reply:
@randallmorey I'm happy to modify this,  please suggest what tag syntax the team is going to use for prod releases?
For now I'm seeing different tags(v*, 1*).
Also please note that our publishing is following semver syntax (tag like this v1.2.3 will be modified to 1.2.3).
We can also add `tag-ignore` field if needed to make sure release is not triggered for some tags .

Reply:
we can maybe do something like this releases/1.2.0 not sure if there is any hashicorp standarts for doing this

Reply:
The main boundary repo uses a `v0.0.0` convention.  We should probably be consistent across the product so that gets my vote.  The point about not being able to do any other release type besides desktop is a good one.  We could do `desktop/v0.0.0*` or similar.